src/frontend/ts/state/chessStore.ts

```diff
@@ -76,6 +79,24 @@ const useChessStore = create<ChessState>((set, get) => ({
       quit: () => {}
     })
   },
+  setLocalArchive: async (data: Array<GameInfo>) => {
+    if (data !== null) {
+      for (const game of data) {
+        const archiveGame: ArchiveGameInfo = {
+          //  fen of the last move
+          position: game.moves[game.moves.length -1].fen,
```

Comment:
Should this be `[game.moves.length - 1]`?

Reply:
good catch. the linter is happy with this too.